src/plugins/timelion/public/directives/_form.scss

```
@@ -34,3 +37,51 @@ select.form-control {
 .fullWidth {
   width: 100%;
 }
+
+.timDropdownWarning {
+  margin-bottom: $euiSize;
+  padding: $euiSizeXS $euiSizeS;
+  color: $euiColorDarkestShade;
+  border-left: solid 2px $timAppWarningColor;
```

I just noticed that the hex for this is the same as our danger color. Just replace with 
```suggestion
  border-left: solid 2px $euiColorDanger;
```